scootapi/server/job_status_test.go

```diff
@@ -1,3 +1,5 @@
+// +build property
```

Comment:
both prop & unit tests in this file split out into two files for this tag to work properly